articles/keywords-and-identifiers.html

```diff
@@ -34,16 +34,17 @@ <h3>Keywords</h3>
 <li>
 	<code>break</code>, <code>case</code>, <code>continue</code>, <code>default</code>, <code>else</code>, <code>fallthrough</code>,
 	<code>for</code>, <code>goto</code>, <code>if</code>, <code>range</code>, <code>return</code>, <code>select</code> and <code>switch</code>
-	are used to control code flows.
+	are used to control flow keywords.
 </li>
 <li>
-	<code>defer</code> and <code>go</code> can also be viewed as control flow keywords.
-	But they are some special. They are modifiers of function calls.
+	<code>defer</code> and <code>go</code> are also control flow keywords, but in a specific manner.
+	They modify function calls, which we'll talk about them in
+	<a href="control-flows-more.html">this article</a>.
 </li>
 </ul>
 
 <p>
-These keywords will be explained detailedly in other articles.
+We will give you a detailed explanation about each keyword in other articles.

```

Comment:
I try to not use the wording "we" (as writers) and "you" (as readers) in Go 101 articles.
These words tend to let readers feel there is distance between readers and writers. ;D

Maybe just change "detailedly" to "in details" is ok?